exercises/armstrong-numbers/.meta/solutions/armstrong_numbers.rb

```diff
@@ -0,0 +1,10 @@
+class ArmstrongNumbers
+  def self.is_valid?(number)
+    digits = number.to_s.split('').map(&:to_i)
```

Comment:
Have you thought about using `chars` here?

Reply:
Well spotted :)